.bumpversion.cfg

```diff
@@ -1,6 +1,6 @@
 [bumpversion]
-current_version = 0.24.0
+current_version = 0.24.1
```

Comment:
Usually we wait until the release PR to bump the version, but I don't mind if you're planning on release right away for testing anyway.

Reply:
I'll hold off